### ISSUE TITLE
latest version of this code has not been released as an add-on

### ISSUE THREAD

**Issue by erluti** [201-created/ember-data-hal-9000#43]
The source for the serializer code has a different normalize method than I'm seeing when I use the debugger to step through the normalization of my model.  It looks like 0.3.1 doesn't have the line that applies transformations but the source here does have it.
Looking at the commit history it seems like it should be there, but I'm still new to the ember and javascript worlds, so I may be misunderstanding something.
Edit: This is the commit that has the transformations applied in the normalize function. https://github.com/201-created/ember-data-hal-9000/commit/d30263c345fd5ab31ff1f7f061f80768c57aa4d8#diff-21df86ecb5f80627fb6dc6057c305faa

**Comment by bschouwerwou** [comment]
That seems to be the case. I'm getting around this in the meantime by updating my `package.json` to have the following: 
```
"ember-data-hal-9000": "git+https://github.com/201-created/ember-data-hal-9000.git#3f751ef07f7e91d122ad7f6ab449e685337e1fdb"
```

**Comment by erluti** [comment]
Thanks, that got me up and running!

**Comment by bantic** [comment]
@erluti I just released version 0.3.2. That should include the commit you link to above. I'm closing this, but please reopen if I missed anything. thanks!
